ui/lib/core/addon/components/form-field-groups.hbs

```diff
@@ -61,7 +61,21 @@
                   @onKeyUp={{@onKeyUp}}
                   @modelValidations={{@modelValidations}}
                   @showHelpText={{@showHelpText}}
-                />
+                >
+                  {{! while identityTokenKey can be set by OSS users, it's only used for the enterprise WIF feature }}
+                  {{#if (and this.version.isEnterprise (eq attr.name "identityTokenKey"))}}
```

Comment:
This feels a little rigid to add this block to `form-field-groups`, instead can we make this more flexible and do something like

```suggestion
                  {{#if (and (has-block) (attr.options.editType "yield"))}}
```

and then instead pass the conditional + yielded component where FormFieldGroups render?